modules/eap-xp-10-env/1.0.0/module.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "eap-xp-10-env"
+description: "JBoss Enterprise Application Platform XP 1.0 env and labels"
```

Comment:
"JBoss Enterprise Application Platform expansion pack 1.0 env and labels"

Reply:
Resolved in https://github.com/jboss-container-images/jboss-eap-7-image/pull/185